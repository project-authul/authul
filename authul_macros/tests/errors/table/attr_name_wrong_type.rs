#[authul_macros::authul_table(42)]
struct Foo;

#[authul_macros::authul_table("name")]
struct Bar;

fn main() {
}
