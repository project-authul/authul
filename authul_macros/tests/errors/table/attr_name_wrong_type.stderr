error: expected ident
 --> tests/errors/table/attr_name_wrong_type.rs:1:31
  |
1 | #[authul_macros::authul_table(42)]
  |                               ^^

error: expected ident
 --> tests/errors/table/attr_name_wrong_type.rs:4:31
  |
4 | #[authul_macros::authul_table("name")]
  |                               ^^^^^^
